data/dfndata/npc/animals.dfn

```diff
@@ -916,4 +927,32 @@ RACE=9
 CONTROLSLOTS=1
 NPCAI=12
 ORIGIN=ml
+}
+
+//crane - Base NPC with shared DFN tags
+[base_crane]
+{
+GET=base_animals
+NAME=#//crane
+ID=0x0fe
+ORIGIN=se
+}
+
+[crane]
+{
+GETSE=crane_se
+}
+
+//gaman - Base NPC with shared DFN tags
+[base_gaman]
+{
+GET=base_animals
+NAME=#//gaman
+ID=0x0f8
+ORIGIN=se
+}
+
+[gaman]
+{
+GETSE=gaman_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=gaman_se
GETT2A=gaman_se
GETUOR=gaman_se
GETTD=gaman_se
GETLBR=gaman_se
GETAOS=gaman_se
GETSE=gaman_se
GETML=gaman_se
GETSA=gaman_se
GETHS=gaman_se
GETTOL=gaman_se